infra-l2-outbound-proxy/src/tests/infra/infra.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Template, Match } from "@aws-cdk/assertions";
+import { testHelper } from "../../lib/test-helper";
+
+const helper = new testHelper();
+
+it("Should contain only one Api Gateway V2 resource definition", () => {
+  helper.getTemplate().resourceCountIs("AWS::ApiGatewayV2::Api", 1);
+})
+
+describe("Outbound Proxy Api Gateway Integration URLs", () => {
+  test.each`
+    ENVIRONMENT      | HMRCURL                                              | PRETTYPROXYURL
+    ${"dev"}         | ${"https://api.isc.externaltest.tax.service.gov.uk"} | ${"proxy.review-bav.dev.account.gov.uk"}
+    ${"build"}       | ${"https://api.isc.externaltest.tax.service.gov.uk"} | ${"proxy.review-bav.build.account.gov.uk"}
+    ${"staging"}     | ${"https://api.isc.externaltest.tax.service.gov.uk"} | ${"proxy.review-bav.staging.account.gov.uk"}
+    ${"integration"} | ${"https://api.isc.externaltest.tax.service.gov.uk"} | ${"proxy.review-bav.integration.account.gov.uk"}
+    ${"production"}  | ${"https://api.isc.externaltest.tax.service.gov.uk"} | ${"proxy.review-bav.account.gov.uk"}
```

Comment:
For prod it would be api.isc.production.tax.service.gov.uk @ViveakV Please correct if its any different.

Reply:
Fixed the production url